client/js/constants.js

```diff
@@ -1,5 +1,24 @@
 "use strict";
 
+const colorCodeMap = [
+	[0, "White"],
```

Comment:
I kinda do for [this](https://github.com/thelounge/lounge/blob/astorije/autocomplete-colors/client/js/lounge.js#L120-L125).
And actually, I'm going to change the first value from a regular int to `"01"`, etc.